ocaml/client_records/records.ml

```diff
@@ -1057,6 +1057,11 @@ let host_record rpc session_id host =
 		let patch_uuids = List.map (fun x -> Client.Pool_patch.get_uuid ~rpc ~session_id ~self:x) patch_refs in
 		patch_uuids
 	in
+	let get_updates () =
```

Comment:
I think I prefer my code for easy to read and also the same coding style as before.